conan.cmake

```diff
@@ -1054,3 +1055,67 @@ macro(conan_config_install)
     message(FATAL_ERROR "Conan config failed='${return_code}'")
   endif()
 endmacro()
+
+
+function(conan_cmake_profile)
+
+  conan_parse_arguments(${ARGV})
```

Comment:
Instead of adding the arguments to the `conan_parse_arguments` macro, you can add the parsing inside this function. Check how the conan_cmake_install() function is implemented: 
https://github.com/conan-io/cmake-conan/blob/835529148ac0def63bbd1d07ceb4e4c130d233af/conan.cmake#L556-L560

Reply:
Done. Please check.